### PR TITLE
[WIP] High level examples document

### PR DESCRIPTION
**Pull request by Lukasa**
It's been suggested that we'd benefit from having a high-level document that talks about how to configure certain topologies in OpenStack. This pull request tracks work on building just such a document.

### CODE REVIEW

File: docs/source/worked-examples-openstack.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+that demonstrate quite complex arrangements. Each example contains diagrams
+that represent a connectivity graph, to add clarity to the textual
+descriptions.
+
+Example 1: Development Machine
+------------------------------
+
+In this example, a user wants to spin up a machine to use as a Linux
+development environment. This user is relatively simple: they want a GUI and
+SSH access, but relatively little else.
```
Comment (alexwlchan) [inline docs/source/worked-examples-openstack.rst]: I think you should remove the second use of "relatively" in this sentence. It feels a little clunky.

File: docs/source/worked-examples-openstack.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+Per the instructions in :doc:`opens-external-conn`, this user cannot create
+Neutron networks or subnets, but they do have access to the networks created
+by the administrator: ``external`` and ``internal``.
+
+Because the user wants to be able to reach the machine from their own laptop,
+they need the machine to be reachable from outside the data center. In
+vanilla Neutron, this would mean provisioning it with a floating IP, but in
+Calico they instead want to make sure the VM is attached to the ``external``
+network. The ``external`` network has the UUID
+``8d5dec25-a6aa-4e18-8706-a51637a428c2``.
```
Comment (ed-harrison) [inline docs/source/worked-examples-openstack.rst]: How would they find this out?  Are they just told by the admins, or can they query it somewhere?

File: docs/source/worked-examples-openstack.rst [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+Before you do, it will be helpful to have a few worked examples for common
+deployment scenarios. In particular, this will make it easy for you to set up
+topologies and examine their connectivity to try to get an understanding of the
+way Calico networks behave.
+
+This document contains several sections, each of which demonstrates a
+representative deployment model for a kind of workload. The early examples are
+simple, containing small numbers of machines, scaling to the later examples
+that demonstrate quite complex arrangements. Each example contains diagrams
+that represent a connectivity graph, to add clarity to the textual
```
Comment (ed-harrison) [inline docs/source/worked-examples-openstack.rst]: Arguably, if you remove this statement about the diagrams, then we could publish this now and just add to it over time.  In other words, I think we have information here, which would already be better published than hidden on github...

**Comment by ed-harrison** [comment]
I think this looks pretty good - clearly we can/should have more examples, but even as it stands I'd like to get it published...

File: docs/source/worked-examples-openstack.rst (hunk 1 above)
Comment (Lukasa) [inline docs/source/worked-examples-openstack.rst]: Good spot. Also, I call the user 'relatively simple', which is a sentence that can definitely be misread!

**Comment by Lukasa** [comment]
Alright, updates made to match both @ed-harrison and @alexwlchan.

**Comment by ed-harrison** [comment]
LGTM

**Pull request merged**